projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -605,6 +605,21 @@ public BatfishLogger getLogger() {
     return _logger;
   }
 
+  @Override
+  public Set<String> getContainerNames() {
```

Comment:
is listContainers not good enough here?

Reply:
listContainers() needs an API key, and only returns containers that are accessible for that key.